schemas/individual_response/definitions.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://eq.ons.gov.uk/individual_response/definitions.json",
+  "individual_response": {
+    "type": "object",
+    "properties": {
+      "enabled": {
+        "type": "boolean",
+        "description": "Controls whether the individual response definition can be shown"
+      },
+      "for_list": {
+        "type": "string",
+        "description": "The list to use when determining whether the individual response definition can be shown"
+      },
+      "individual_section_id": {
+        "type": "string",
+        "description": "The id of the individual section for which an individual response can be requested"
+      }
```

Comment:
Thinking about it, I can't see why we want to keep defining these for each block. These should not be different within 1 questionnaire, therefore these seem like top level properties of the hub and blocks should only needed to specify whether IR is enabled or not.